content/400-reference/100-system-requirements.mdx

```diff
@@ -52,6 +52,10 @@ Note that systems that can run the supported versions of Node.js will most likel
 
 On Windows [Microsoft Visual C++ Redistributable 2015](https://download.microsoft.com/download/9/3/F/93FCF1E7-E6A4-478B-96E7-D4B285925B00/vc_redist.x64.exe) or newer must be installed (which is by default the case on most modern installations).
 
+#### macOS runtime dependencies
+
+Prisma supports macOS 10.15 or newer on Macs with Intel CPUs and macOS 11 or newer on Macs with Apple Silicon chips. There are no additional platform-specific requirements on macOS other than what is listed for all platforms in the [Software requirements](#software-requirements) section.
```

Comment:
Why the difference depending on Intel/Apple silicon?

Reply:
macOS 11 is the minimal version that runs on Apple Silicon, we can't target 10.15 there

Reply:
I you can not run older than 11 on Apple Silicon, can this then not completely be left out as it is implied in owning a Apple Silicon laptop that you can not run older OS?
```suggestion
Prisma supports macOS 10.15 or newer. There are no additional platform-specific requirements on macOS other than what is listed for all platforms in the [Software requirements](#software-requirements) section.
```